source/components/organisms/SummaryList/SummaryList.tsx

```diff
@@ -163,7 +163,7 @@ const SummaryList: React.FC<Props> = ({
     return typeof answer !== 'undefined';
   });
   itemsWithAnswers
-    .forEach((item) => {
+    .forEach(( item) => {
```

Comment:
Lots of small white-space changes in this file. Is this intentional? This one for example is inconsistent with the rest.

Reply:
Not intentional. They wore added when I saved file, somethings fishy.

Reply:
Removed whitespaces. Don't know why spaces were left after save, all spaces should be removed before pushing code.